src/plugins/intel_gpu/src/graph/impls/onednn/utils.cpp

```diff
@@ -204,13 +204,13 @@ dnnl::memory::format_tag convert_data_format(cldnn::format fmt) {
 }
 
 void combine_bf_with_first_spatial_dim(cldnn::layout& l) {
-    auto pshape = l.get_shape();
+    auto pshape = l.get_partial_shape();
     ov::Shape new_shape{1, 1};
     for (size_t i = 0; i < pshape.size(); ++i) {
         if (i < 2) {
-            new_shape[0] *= pshape[i];
+            new_shape[0] *= pshape[i].is_dynamic() ? 1 : pshape[i].get_length();
```

Comment:
I think `combine_bf_with_first_spatial_dim()` should always run in static, Are't you?

Reply:
@ahnyoung-paul  shouldn't we do like 
1) combine_bf.. is called for static shape only 
2) add assert if the layout is dynamic
?

Reply:
change the code to "new_shape[0] *= pshape[i].get_length();" and do not add assert to the code because get_length() has assert for dynamic dimension.